subsys/net/ip/tcp.c

```diff
@@ -273,9 +273,11 @@ struct net_tcp *net_tcp_alloc(struct net_context *context)
 	tcp_context[i].state = NET_TCP_CLOSED;
 	tcp_context[i].context = context;
 
-	tcp_context[i].send_seq = tcp_init_isn();
-	tcp_context[i].recv_wnd = min(NET_TCP_MAX_WIN, NET_TCP_BUF_MAX_LEN);
-	tcp_context[i].send_mss = NET_TCP_DEFAULT_MSS;
+	tcp_context[i].send_seq      = tcp_init_isn();
+	tcp_context[i].recv_wnd      = min(NET_TCP_MAX_WIN,
```

Comment:
Zephyr codestyle doesn't requires this equal sign alignment, you can see it by 2 lines above the affected hunk. It's especially true for Zephyr codestyle, which uses small line length, and big indents, so a line can start to wrap easily, just like happens here.

As a general rule of thumb, the less size of a patch (== the less changes it makes), the better.